aot-tests/pom.xml

```diff
@@ -157,6 +156,8 @@
           <excludedUnlinkableWasts/>
           <excludedUninstantiableWasts/>
           <excludedWasts>
+            <!-- obsolete-keywords is only stressing the wat parser -->
```

Comment:
Should we make this part of the Maven plugin?

Reply:
It's a single exception at the moment, I'll wait to have more cases to refactor in the Maven Plugin.